src/docs/docs/getting-started/vscode-ssh.md

```diff
@@ -309,7 +238,9 @@ sh ./tools/aws-ip.sh
 pnpm install
 ```
 
-2. Start all Telescope services. This will take some time to complete
+2. Start all Telescope services using the environment variables set in `.env`. This will take some time to complete
+
+Note: Do not use `pnpm services:start`. This will use the environment variables in `config/env.development` and we don't want that here.
```

Comment:
We should fix that in a follow-up.  We probably need to create some configs for running within AWS.

Reply:
This is related to #3017 , we can add on to it or file a new issue. I'd like to see how the team resolves #3017 so I will follow it closely.